session_csrf/__init__.py

```diff
@@ -32,7 +37,14 @@ def prep_key(key):
     prefixed = PREFIX + key
     return hashlib.md5(prefixed).hexdigest()
 
-class CsrfMiddleware(object):
+
+def is_user_authenticated(request):
+    if DJANGO_VERSION < (1, 10, 0):
+        return request.user.is_authenticated()
+    else:
+        return request.user.is_authenticated
+
+class CsrfMiddleware(deprecation.MiddlewareMixin if DJANGO_VERSION >= (1, 10, 0) else object):
```

Comment:
I'm actually not familiar with what `deprecation.MiddlewareMixin` but I feel like this class definition now deserves a comment that explains why we do what we do here.